### DISCUSSION TITLE
Shrink portable Windows distribution only to modify DDS files

### DISCUSSION THREAD

**Discussion by SerVB** [ImageMagick/ImageMagick#2411] [General] [answered]
Hi!

I see there are many files in the `ImageMagick-7.0.10-27-portable-Q8-x86.zip` distribution:

![image](https://user-images.githubusercontent.com/20105593/90186182-708d7700-ddc0-11ea-84fc-3de2355b09b4.png)

All I want is to clear areas and draw some PNG images on a DDS image and save it as another DDS image. Like this:

```
convert sourceBattleAtlas.dds \
-alpha set -region 20x20+5+5 -alpha transparent -draw 'image srcOver 0,0 20,20 pathToTexture1.png' \
-alpha set -region 20x20+5+25 -alpha transparent -draw 'image srcOver 0,0 20,20 pathToTextrure2.png' \
battleAtlas.dds
```

Are some files from the distribution not needed for this task? If they can be deleted, it will help to reduce the size!

**Comment by snibgo** [comment, marked as answer]
For IM v7, I suggest you use "magick", not "convert". I suppose you don't need any other *.exe files.

You don't need *.txt or *.rdf or ChangeLog or sRGB.icc.

You _do_ need *.xml.
